### PR TITLE
Cache Requests library & use for ReportingOrganisation.json & Organis…

### PR DESCRIPTION
**Pull request by ghost**
…ationRegistrationAgency.json

Previously these resources were fetched every time we processed something

### CODE REVIEW

**Comment by ghost** [comment]
Pretty sure this is actually doing something :-)

If you stick into settings.py:

```
import logging
logging.getLogger('requests_cache').setLevel('DEBUG')
```

clear out the cache directory, run a server and process some files .... the first file processed will drop some "Pre-cache checks for response from " lines with all disabled flags false. Subsequent files processed will show no log lines

**Comment by ghost** [comment]
And test expiration - leave logging statements in, change expiration to 3 mins, start server, process a file, see it getting file with "Pre-cache checks for response from" lines, process more files, see cache being used by lack of log lines, wait 3 minutes, process another file - see log lines of "Expired response; attempting to re-send request" and "Pre-cache checks for response from"

File: cove_iati/settings.py [hunk 1]
```diff
@@ -107,3 +110,18 @@
...
+# Note on deploying new versions of this app the cache directory should be cleared
+# as new versions of requests & requests_cache libraries may use different versions
+# https://requests-cache.readthedocs.io/en/latest/user_guide/troubleshooting.html#potential-issues
+REQUESTS_SESSION_WITH_CACHING = CachedSession(
+    'iati_cove_cache',
+    backend=FileCache(
+        cache_name=env('REQUESTS_CACHE_DIR')
+    ),
+    expire_after=timedelta(days=1),  # Expire responses after one day
+    allowable_methods=['GET'],       # Cache POST requests to avoid sending the same data twice
```
Comment (Bjwebb) [inline cove_iati/settings.py]: This comment is out of sync with what's set?

Comment (ghost) [reply cove_iati/settings.py]: Fixed

**Review by Bjwebb — APPROVED** [review]

**Review by Bjwebb — APPROVED** [review]

**Pull request merged**